fastlane/Fastfile

```diff
@@ -48,11 +48,11 @@ platform :ios do
       scheme: "RevenueCat",
       testplan: "Coverage",
       prelaunch_simulator: true,
-      output_types: '',
       fail_build: false,
-      result_bundle: true
+      result_bundle: true,
+      number_of_retries: 5,
```

Comment:
I feel like 5 might be too much? If there's an actual failure a PR might clog CircleCI for like half an hour waiting for it to pass. I think 2 should cover flaky tests. They've been way less flaky lately.

Reply:
It only retries the individual test that fails during the same run so it won't be _terribly_ slow 😅  But how does 3 sound?

Reply:
Oooh in that case 5 is fine. I thought it would run the whole build again.

Reply:
Ah, nope nope! This new Xcode 13 way is so good 🙂